std_msgs/Point32 frame_p1
std_msgs/Point32 frame_p2
---
std_msgs/Point32 door_p1
std_msgs/Point32 door_p2
float32 height
std_msgs/Point32 handle
